content/kubeone/master/guides/local/_index.en.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Local subcommand"
+date = 2022-08-05T15:58:00+02:00
+enableToc = true
++++
+
+## Introduction
+
+`kubeone local` subcommands helps you to initialize a **local** (e.g. on the
+local machine, in the current shell), 1 node all-in-one cluster.
+
+## Use Cases
+
+Such a setup could be useful for cases such as:
+* Single edge server
+* Developer VM (kinda competing with kind but without placing everything in
+  container)
+* CI VM
+
+## Prerequisite
+
+Local user that run the `kubeone local` ether has to have passwordless `sudo` or
+be a `root` user.
+
+### How to configure passwordless sudo
+
+Edit the `/etc/sudoers` file with:
+```shell
+sudo visudo
+```
+
+Add a line:
+```
+<your_username> ALL=(ALL:ALL) NOPASSWD:ALL
+```
+Replacing `<your_username>` with your login (i.e. `$USER`).
```

Comment:
I guess you can't put just `$USER`, so it's probably more correct to say the value of `$USER`.

Reply:
done